### PR TITLE
Updated member pics.

### PR DESCRIPTION
**Pull request by revagomes**
- Handrus, Mario and Denis pics removed.
- Marcos pic added.
- Started the loop to complete the members mosaic.

### CODE REVIEW

File: src/js/lib/photo-swipe.js [hunk 1]
```diff
@@ -20,12 +19,13 @@ const members = new Vue({
...
       { name: 'celso', image: '/images/members/9.jpg' },
       { name: 'becker', image: '/images/members/10.jpg' },
-      { name: 'mario', image: '/images/members/11.jpg' },
       { name: 'sebas', image: '/images/members/12.jpg' },
       { name: 'ban', image: '/images/members/13.jpg' },
       { name: 'rafa', image: '/images/members/14.jpg' },
-      { name: 'denis', image: '/images/members/15.jpg' },
       { name: 'helal', image: '/images/members/16.jpg' },
+      { name: 'marcos', image: '/images/members/17.jpg' },
+      { name: 'recidive', image: '/images/members/1.jpg' },
```
Comment (lucasconstantino) [inline src/js/lib/photo-swipe.js]: Why is there repeated "recidive" and "italo" members?

Comment (revagomes) [reply src/js/lib/photo-swipe.js]: It was just to complete the mosaic.

Comment (lucasconstantino) [reply src/js/lib/photo-swipe.js]: And is it really needed? As far as I understand it, the current mosaic is quite flexible. We may need to adjust a bit of css, but that's all.

Comment (revagomes) [reply src/js/lib/photo-swipe.js]: It's true. In fact I've already updated the code to remove the duplicated and styles.
It was also manually merged into stage. ;)

**Pull request closed without merging**